ovn-tester/ovn_tester.py

```diff
@@ -304,25 +327,39 @@ def run_test_netpol_multitenant(ovn, global_cfg, cfg):
         for ns in all_ns:
             ns.unprovision()
 
+def cluster_density_run(ovn, burst=1):
+    ns = [
+        Namespace(ovn, 'NS_'.join(random.choice(string.ascii_lowercase)))
+        for _ in range(burst)
+    ]
+    # create 6 short lived "build" pods
+    build_ports = ovn.provision_ports(6*burst)
+    for i in range(burst):
+        ns[i].add_ports(build_ports[6*i : 6*i + 5])
+    ovn.ping_ports(build_ports)
+    # create 4 legacy pods
+    ports = ovn.provision_ports(4*burst)
+    for i in range(burst):
+        ns[i].add_ports(ports[4*i : 4*i + 3])
+        # add VIPs and backends to cluster load-balancer
+        ovn.provision_vips_to_load_balancers(ports[i*4 : i*4 + 1])
+        ovn.provision_vips_to_load_balancers([ports[i*4 + 2]])
+        ovn.provision_vips_to_load_balancers([ports[i*4 + 3]])
```

Comment:
Too many magic numbers in this function now.  Maybe we should add some constant definitions, e.g., `DENSITY_N_BUILD_PODS = 6`, `DENSITY_N_PODS = 4`.